proposals/stage-1/CODE_OF_CONDUCT/HANDLING-REPORTS.md

```diff
@@ -0,0 +1,88 @@
+# Handling reports and escalation
+This document describes the proposed process for handling reports and escalation, along with relevant roles in the OpenJS Foundation.
+
+This process covers two types of reports based on ./FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md:
+
+* Reports for spaces managed by the Cross Project Council (CPC) which come in through `report@opensjsf.org`
+* Escalations which come in through `coc-escalation@openjsf.org`.   
+
+Generally, reports to coc-escalation should refer to a report that was already submitted to:
+
+* report@openjsf.org or
+* one of the projects reporting emails 
+* explanation of why the reportee does not feel comfortable using the previous reporting emails for the initial report
+
+If a report to coc-esclation does not include one of those, the reportee will be asked to to provide one of those three as the 
+reason for reporting to coc-escalation.
+
+## Confidentiality and record-keeping
+Personal information is confidential. All reports should be recorded, together with the discussion of it. The following private repos will be used to record and discuss reports.
+
+* report@openjsf.org -> github.com/openjs-foundation/moderation
+* coc-escalation@openjsf.org -> github.com/openjs-foundation/coc-escalation
+
+For each report there should be an issue which captures the report, discussion and final conclusion.  The report should NOT contain identifying details of the reporter.
+
+## Conflict of Interest
+
+Any member of the CPC or CoCP who is involved in the report should recuse themselves from the discussions.
```

Comment:
This is the first place CoCP is used in this document so it should probably be written out at Code of Conduct Panel (CoCP)

Reply:
fixed